build.config.ts

```diff
@@ -2,10 +2,12 @@ import { defineBuildConfig } from 'unbuild'
 
 export default defineBuildConfig({
   declaration: true,
-  emitCJS: false,
   entries: [
     'src/index',
     'src/node',
     'src/undici'
-  ]
+  ],
+  rollup: {
+    emitCJS: true
```

Comment:
CJS support is limited and via a manual entry (https://github.com/unjs/ohmyfetch/tree/main/cjs). We cannot have `require('node-fetch')`. You can drop both options.

Reply:
I see. Thanks for the explanation.